src/rotational.jl

```diff
@@ -1,17 +1,13 @@
 export
     RotDiffusionProcess,
-    MultiRotationState,
-    features
+    MultiRotationState
```

Comment:
Do we need to export this? The user shouldn't see this type.

Reply:
I don't think we need to. I'll remove it.